src/cli/playground/lib/index.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+// Given a function `f` that may return a Promise or a value, `reify` returns
+// another function, such that upon being called, any Promise instance returned
+// by `f` will be resolved and printed to the console and undefined will be
+// returned. Any other value will be returned to the caller.
+function reify(f/*: (...args: Array<mixed>) => mixed*/)/*: mixed*/ {
+  return function () {
+    const ret = f.apply(this, arguments);
+    if (ret instanceof Promise) {
+      ret.then(result => console.log(`[Fulfilled]\n`, result))
+         .catch(err => console.error(err));
+      return undefined;
+    }
+    return ret;
+  };
+}
+
+function methods(controller/*: Object*/, methods/*: Array<string>*/)/*: {[methodName: string]: Function}*/ {
+  const loaded = { };
+  for (const method of methods) {
```

Comment:
The object referenced by `loaded` is modified, but the reference `loaded` itself isn't changed, hence `const` is appropriate here; I'll admit that `let` indicates that the underlying object will be mutated.